k8s/perftest/common/divorce/div-emca.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: div-emca
+  namespace: divorce
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: div-emca
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: div-emca
+    version: 1.1.0
+  values:
+    rollback:
+      enable: true
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "a2e12f21-e77a-4b9f-91f3-6e7b3867892e"
+      image: hmctspublic.azurecr.io/div/emca:latest
+      ingressHost: div-emca-perftest.service.core-compute-perftest.internal
+      environment:
+        AUTH_PROVIDER_SERVICE_CLIENT_MICROSERVICE: divorce_document_generator
+        EVIDENCE_MANAGEMENT_HEALTH_URL: http://dm-store-perftest.service.core-compute-perftest.internal/health
+        EVIDENCE_MANAGEMENT_UPLOAD_FILE_URL: http://dm-store-perftest.service.core-compute-perftest.internal/documents
+        DOCUMENT_MANAGEMENT_STORE_URL: http://dm-store-perftest.service.core-compute-perftest.internal
+        AZURE_APPLICATIONINSIGHTS_INSTRUMENTATIONKEY: 63c30ef9-a27b-4d38-8afc-4c180fb6ac33
```

Comment:
this is conflicting https://github.com/hmcts/cnp-flux-config/pull/468/files#diff-014abc7772a2fdc96938d968da5874dbR22